sklearn/feature_extraction/tests/test_text.py

```diff
@@ -1097,3 +1097,28 @@ def test_vectorizers_invalid_ngram_range(vec):
     if isinstance(vec, HashingVectorizer):
         assert_raise_message(
             ValueError, message, vec.transform, ["good news everyone"])
+
+
+@pytest.mark.parametrize("vec", [CountVectorizer()])
+def test_countvectorizer_sort_features_64bit_sparse_indices(vec):
+    # If a count vectorizer has to store >= 2**31 count values, the sparse
+    # storage matrix has 64bit indices / indptrs.  This requires ~2*8*2**31
+    # bytes of memory in practice, so we just test the method that would
+    # hypothetically fail.
```

Comment:
Maybe just say,
"""
Check that `_sort_features` preserves the indices dtype of sparse arrays with 64 bit indices
"""

Reply:
Yep, I'll update it to something cleaner.